packages/openapi3/src/lib.ts

```diff
@@ -54,6 +54,14 @@ export interface OpenAPI3EmitterOptions {
    * @default "never"
    */
   "include-x-typespec-name"?: "inline-only" | "never";
+
+  /**
+   * How to handle safeint type. Options are:
```

Comment:
should likely add a blurb about the default:  if not specified, 'int64' will be used.